tests/Integration/Behaviour/Features/Scenario/Discount/BO/add_cart_level_discount.feature

```diff
@@ -0,0 +1,125 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s discount --tags add-cart-level-discount
+@restore-all-tables-before-feature
+@restore-languages-after-feature
+@add-cart-level-discount
+Feature: Add discount
+  PrestaShop allows BO users to create discounts
+  As a BO user
+  I must be able to create discounts
+
+  Background:
+    Given shop "shop1" with name "test_shop" exists
+    Given there is a currency named "usd" with iso code "USD" and exchange rate of 0.92
+    Given there is a currency named "chf" with iso code "CHF" and exchange rate of 1.25
+    Given currency "usd" is the default one
+    And language with iso code "en" is the default one
+    And language "french" with locale "fr-FR" exists
+
+  Scenario: Create a simple cart level discount
+    When I create a cart level discount "basic_cart_level_discount"
+    And discount "basic_cart_level_discount" should have the following properties:
+      | free_shipping | false |
```

Comment:
You should at least check that the type is the right one (and in the other creation steps as well)